src/firstboot/firstboot.c

```diff
@@ -97,8 +97,7 @@ static bool press_any_key(void) {
         char k = 0;
         bool need_nl = true;
 
-        printf("-- Press any key to proceed --");
-        fflush(stdout);
+        printf("-- Press any key to proceed --\n");
```

Comment:
doesn't matter, but maybe we should convert this to puts(). after all we are not using a format string here, and you could drop the the `\n` then